### PR TITLE
[4.0]improved contrast in icon during installation

### PR DESCRIPTION
**Pull request by bahl24**
Pull Request for Issue - Improved contrast and consistency for 'warning' label.

### Testing
run node build.js --compile-css

### Summary of Changes
Changed text color

### Expected result
Screenshot from 2019-03-17 05-44-39

### Actual result
Screenshot from 2019-03-17 06-03-23

Screenshot from 2019-03-17 09-24-28

### CODE REVIEW

**Comment by chmst** [comment]
@bahl24 good idea! But it is better not to use a color definition. Could you use the scss variable white instead?

**Comment by bahl24** [comment]
@chmst I have changed, but I was wondering why is white better than its hex code?

**Comment by chmst** [comment]
Indeed, at this place it makes no difference, as the background here also is white and probably will not be changed. So thank you - I will test this asap.

File: installation/template/scss/template.scss [hunk 1]
```diff
@@ -279,7 +279,7 @@ caption {
   caption-side: top;
 }
 .badge-warning {
-  color: #292b2c;
+  color: white;
```
Comment (Quy) [inline installation/template/scss/template.scss]: It should be `$white` like line 50.

**Comment by bahl24** [comment]
@Quy Done

**Comment by Quy** [comment]
Actually lines 281-283 can be removed since the same class definition is defined starting at line 3420 in the generated `template.css`.

```
.badge-warning {
  color: #fff;
  background-color: #b35900; }
  a.badge-warning:hover, a.badge-warning:focus {
    color: #fff;
    background-color: #804000; }
  a.badge-warning:focus, a.badge-warning.focus {
    outline: 0;
    box-shadow: 0 0 0 0.2rem rgba(179, 89, 0, 0.5); }
```

**Comment by bahl24** [comment]
> Actually lines 281-283 can be removed since the same class definition is defined starting at line 3420 in the generated `template.css`.
> 
> ```
> .badge-warning {
>   color: #fff;
>   background-color: #b35900; }
>   a.badge-warning:hover, a.badge-warning:focus {
>     color: #fff;
>     background-color: #804000; }
>   a.badge-warning:focus, a.badge-warning.focus {
>     outline: 0;
>     box-shadow: 0 0 0 0.2rem rgba(179, 89, 0, 0.5); }
> ```

Done

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on f2b231c7b1525d65cbb85cc13592fb363c462fd1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24213">issues.joomla.org/tracker/joomla-cms/24213</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on f2b231c7b1525d65cbb85cc13592fb363c462fd1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24213">issues.joomla.org/tracker/joomla-cms/24213</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24213">issues.joomla.org/tracker/joomla-cms/24213</a>.</sub>

**Pull request merged**